src/core/env/uat/backend.tfvars

```diff
@@ -1,4 +1,4 @@
 resource_group_name  = "io-infra-rg"
-storage_account_name = "pagopainfraterraformuat"
+storage_account_name = "selcustinfraterraform"
 container_name       = "azureadstate"
```

Comment:
Should be **azurerm** for all

Reply:
Actual resource name used for UAT is selcustinfraterraform, as created during authorizing process.
Should we first try to manually rename it inside the subscription?

Reply:
Sorry container name